rts/Rendering/GlobalRendering.cpp

```diff
@@ -656,6 +657,7 @@ void CGlobalRendering::SwapBuffers(bool allowSwapBuffers, bool clearErrors)
 		glBindFramebuffer(GL_READ_FRAMEBUFFER_EXT, 0);
 
 		SDL_GL_SwapWindow(sdlWindow);
+		FrameMark;
```

Comment:
I'm slightly unhappy with the way tracy author specifies macros. I believe the de-facto standard is to capitalize and underscore. Unsure if we should do `#define TRACY_FRAMEMARK FrameMark` or something like that.

Reply:
@marcushutchings opinions? If you also agree, I can refactor out into our own header with all tracy's exported defines re-declared to keep consistency.

Reply:
I'll point out that the previous line, SDL_GL_SwapWindow, also doesn't match our coding style. But I doubt there will be a call to change that. :)

First we should understand why it is that way. From what I can tell the approach was to minimize the visual impact on the code - after all tracing is one of those things (like logging) that distracts the reader from comprehending the purpose of the code contributing to the program's main objective.

With this suggestion, tracing points stand out a lot more so that will mean programmers will be more likely to put blank lines around them to separate them from the normal code execution lines. Thus it will make it clear that these are related to performance tracing rather than potentially being mistaken for being related to the normal execution of code, and makes it easier to keep track on how and where we are placing tracing.

I see the benefit both ways on this. It is good point to bring up and this is the time to think about it (before the first merge.)

I like the suggestion to wrap these macros and I think the example `TRACY_FRAMEMARK` is right on the money for this approach. It also fits with the logging macros. Unless I'm mistaken, it doesn't look like a lot of work to switch over at this stage.

That all said, I'm open to be persuaded otherwise via other arguments and counter arguments.

Reply:
The main con in my opinion is maintenance when we will update versions of the library and the fact that you have to do additional hop and translation when you let's say read tracy doc and then want to use that, you first have to lookup "how does the macro look in our code", that also works in the other direction.

Do you also want to rewrite all macros, or only the ones you use, and deal with it as you go.